src/microbe_stage/Microbe.Contact.cs

```diff
@@ -548,18 +557,20 @@ public void Kill()
             var sceneToUse = new ChunkConfiguration.ChunkScene();
 
             // Try all organelles in random order and use the first one with a scene for model
-            foreach (var organelle in organelles.OrderBy(_ => random.Next()))
+            foreach (var organelle in organellesAvailable.OrderBy(_ => random.Next()))
             {
                 if (!string.IsNullOrEmpty(organelle.Definition.CorpseChunkScene))
                 {
                     sceneToUse.LoadedScene = organelle.Definition.LoadedCorpseChunkScene;
+                    organellesAvailable.Remove(organelle);
                     break;
                 }
 
                 if (!string.IsNullOrEmpty(organelle.Definition.DisplayScene))
                 {
                     sceneToUse.LoadedScene = organelle.Definition.LoadedScene;
                     sceneToUse.SceneModelPath = organelle.Definition.DisplaySceneModelPath;
+                    organellesAvailable.Remove(organelle);
```

Comment:
If this is ever hit and the loop runs again, won't this actually hit an exception and not be able to continue looping?

Reply:
It's a list. It's fine.